docs/standard/base-types/best-practices-regex.md

```diff
@@ -100,59 +100,43 @@ By default, the last 15 most recently used static regular expression patterns ar
+To summarize, we recommend that you:
 
-The following example compares the performance of compiled and interpreted regular expressions when reading the first 10 sentences and when reading all the sentences in the text of Theodore Dreiser's *The Financier*. As the output from the example shows, when only 10 calls are made to regular expression matching methods, an interpreted regular expression offers better performance than a compiled regular expression. However, a compiled regular expression offers better performance when a large number of calls (in this case, over 13,000) are made.
+- Use *interpreted* regular expressions when you call regular expression methods with a specific regular expression relatively infrequently.
+- Use *source-generated* regular expressions if you're using `Regex` in C# with arguments known at compile time, and you're using a specific regular expression relatively frequently.
+- Use *compiled* regular expressions when you call regular expression methods with a specific regular expression relatively frequently and you're using .NET 6 or an earlier version.
 
-[!code-csharp[Conceptual.RegularExpressions.BestPractices#5](../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.regularexpressions.bestpractices/cs/compare1.cs#5)]
-[!code-vb[Conceptual.RegularExpressions.BestPractices#5](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.regularexpressions.bestpractices/vb/compare1.vb#5)]
+It's difficult to determine the exact threshold at which the slower execution speeds of interpreted regular expressions outweigh gains from their reduced startup time, or the threshold at which the slower startup times of source-generated or compiled regular expressions outweigh gains from their faster execution speeds. It depends on various factors, including the complexity of the regular expression and the specific data that it processes. To determine which regular expressions offer the best performance for your particular application scenario, you can use the <xref:System.Diagnostics.Stopwatch> class to compare their execution times.
```

Comment:
This sentence is rather hard to read, it feels really long: "It's difficult to determine the exact threshold at which the slower execution speeds of interpreted regular expressions outweigh gains from their reduced startup time, or the threshold at which the slower startup times of source-generated or compiled regular expressions outweigh gains from their faster execution speeds" But I'm not sure how to best rephrase it.